src/frontend/components/DashboardConnected/DashboardConnected.tsx

```diff
@@ -0,0 +1,48 @@
+import { connect } from 'react-redux';
+import { Dispatch } from 'redux';
+
+import { addResource } from '../../data/genericReducers/resourceById/actions';
+import { RootState } from '../../data/rootReducer';
+import { modelName } from '../../types/models';
+import { Video } from '../../types/Video';
+import { Dashboard } from '../Dashboard/Dashboard';
+
+/** Props shape for the DashboardConnected component. */
+interface DashboardConnectedProps {
+  video: Video;
+}
+
+/**
+ * Replace the (read-only) video from context with one from the resources part of the
+ * state if available as it will hold the most recent version.
+ * Also, just pass the jwt along.
+ */
+export const mapStateToProps = (
+  state: RootState,
+  { video }: DashboardConnectedProps,
+) => ({
+  jwt: state && state.context && state.context.jwt,
+  video:
+    (state &&
+      state.resources &&
+      state.resources[modelName.VIDEOS] &&
```

Comment:
The two lines before are useless, you can remove them and update the tests according to this change.